slick/src/test/scala/za/co/absa/fadb/slick/FaDbPostgresProfileSuite.scala

```diff
@@ -108,14 +109,14 @@ class FaDbPostgresProfileSuite extends AsyncFlatSpec {
       InetString("168.0.0.1"),
       MacAddrString("12:34:56:78:90:ab")
     )
-    // because postgres does not fully support time zone as Java, so we need to clear it for later successful assert
+    // because postgres does not fully support time zone as Java, so we need to clear test for later successful assert
```

Comment:
wrong renaming, keep the original comment - maybe it did it automatically

Reply:
Fixed in commit - 7ecd479dc774ce11dd3166282fdd013c56cb35a7.